### ISSUE TITLE
Can't solve graphs when data is included.

### ISSUE THREAD

**Issue by GearsAD** [GearsAD/GraffSDK.jl#104]
```
2019-03-05 22:44:23┌ Info: ERROR
2019-03-05 22:44:23│ KeyError: key "sourceParams" not found
2019-03-05 22:44:23│ Stacktrace:
2019-03-05 22:44:23│ [1] CloudGraphs.BigDataElement(::Dict{String,Any}, ::String) at ./dict.jl:478
2019-03-05 22:44:23│ [2] neoNode2CloudVertex(::CloudGraphs.CloudGraph, ::Neo4j.Node) at /root/.julia/dev/CloudGraphs/src/CloudGraphs.jl:163
```

**Comment by GearsAD** [comment]
"SurveyBotfcb17a" is one of these - this is why it is not solving.

**Comment by GearsAD** [comment]
#105

**Comment by GearsAD** [comment]
Fix for #104 is in https://github.com/GearsAD/GraffSDK.jl/releases/tag/v0.4.5
